operator/roles/forkliftcontroller/templates/controller/deployment-controller.yml.j2

```diff
@@ -76,6 +76,10 @@ spec:
         - name: CDI_EXPORT_TOKEN_TTL
           value: "{{ CDI_EXPORT_TOKEN_TTL }}"
```

Comment:
@bennyz might be a bug here? shouldn't it be `controller_cdi_export_token_ttl`?

Reply:
yes

Reply:
done in a new commit